docs/requirements.txt

```diff
@@ -1,4 +0,0 @@
-ipython_genutils
```

Comment:
This file is required for [ammaraskar/sphinx-action@master](https://github.com/ammaraskar/sphinx-action#how-to-use). 

"If you have any Python dependencies that your project needs (themes, build tools, etc) then place them in a requirements.txt file inside your docs folder."